packages/webapp/src/constants/chat.ts

```diff
@@ -12,10 +12,12 @@ const dateString =
 
 // default system message obtained using the following method: https://twitter.com/DeminDimin/status/1619935545144279040
 export const _defaultSystemMessage =
-  import.meta.env.VITE_DEFAULT_SYSTEM_MESSAGE ??
-  `You are ChatGPT, a large language model trained by OpenAI.
-Carefully heed the user's instructions. 
```

Comment:
Do you mean: "Carefully read the user's instructions."?

I don't know what heed means

Reply:
It was removed. That was the prompt came with BetterChatGPT